src/nidcpower/metadata/functions.py

```diff
@@ -2949,6 +2956,7 @@
         'method_templates': [
             {
                 'documentation_filename': 'default_method',
+                'library_interpreter_filename': 'none',
```

Comment:
I must be missing something. Shouldn't this prevent us from adding a method to the LibraryInterpreter?
But it looks like we added one: https://github.com/ni/nimi-python/blob/7f9d9993ef60f7315f36ee08385b4b2e35114ef6/generated/nidcpower/nidcpower/_library_interpreter.py#L380

Reply:
Different function, though named almost the same:
GetLastSelfCalLastDateAndTime
GetSelfCalLastDateAndTime
(The former is an invented python-only wrapper around the latter, used for datetime conversion.)